components/browse/ContentCardSection.vue

```diff
@@ -57,6 +55,31 @@
           return card;
         });
       }
+    },
+    methods: {
+      moreLabelKey: function (card) {
```

Comment:
When we end up having many methods all taking the same card parameter, it points to the need for a new component scoped to one of those cards.

Reply:
yes, certainly something we could refactor by adding these values to contentful. Let's do so later when we have more of the cards ready, including entity cards and potential changes coming out of the Open Graph work.